runtime/src/main/scala/akka/grpc/internal/NettyClientUtils.scala

```diff
@@ -42,33 +44,39 @@ object NettyClientUtils {
     builder.build
   }
 
-  // FIXME couldn't we use the inputstream based trustManager() method instead of going via filesystem?
-  private def loadCert(name: String): File = {
```

Comment:
As suggested in the FIXME, this code could be removed because the Netty API supports reading from an `InputStream` as well as a `File`.